packages/components/src/components/SelectMenu/selectMenuStyles.ts

```diff
@@ -0,0 +1,31 @@
+// selectMenuStyles.ts
+import { EDSStyleSheet } from "../../styling";
+
+export const selectMenuStyles = EDSStyleSheet.create((theme, props: { menuOpen: boolean }) => {
+    const { menuOpen } = props;
+
+    return {
+        contentContainer: {
```

Comment:
I see that both components dont have a maxheight, this might cause an issue when the list of options is very long

Reply:
Also maybe they should be scrollable?

Reply:
Yes, i agree (^;